templates/golang/files/.gitignore

```diff
@@ -1,2 +1,3 @@
 /.GOPATH
 /bin
+.platform/local/*
```

Comment:
Shouldn't the local directory have its own .gitignore in it that works?  This shouldn't be necessary.  (Let's figure out why it was necessary and make it not necessary.)